app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_people = [
+        Person(person["name"], person["age"]) for person in people
+    ]
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        if person.get("husband"):
+            Person.people[person["name"]].husband = Person.people[
+                person["husband"]
+            ]
```

Comment:
Since you already created people instances in class u could just remove one of redundant if, by making code like this(according to DRY):
```
for person in people:
        if person.get("wife"):
            wife = Person.people[person.get("wife")]
            husband = Person.people[person.get("name")]
            husband.wife = wife
            wife.husband = husband
```